actor/todo/src/lib.rs

```diff
@@ -0,0 +1,282 @@
+use std::collections::HashMap;
+
+use anyhow::{anyhow, Result};
+use log::{info, trace, warn};
+use serde::{Deserialize, Serialize};
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse, HttpServer, HttpServerReceiver};
+use wasmcloud_interface_keyvalue::{
+    IncrementRequest, KeyValue, KeyValueSender, SetAddRequest, SetDelRequest, SetRequest,
+};
+#[derive(Debug, Default, Actor, HealthResponder)]
+#[services(Actor, HttpServer)]
+struct TodoActor {}
+
+// FIXME: contribute this upstream
```

Comment:
specifically in here: https://github.com/wasmCloud/interfaces/blob/main/httpserver/rust/src/lib.rs

Reply:
https://github.com/wasmCloud/interfaces/pull/10